hyperlane/relayer_runner.go

```diff
@@ -83,6 +83,40 @@ func (r *RelayerRunner) Run(ctx context.Context) error {
 					continue
 				}
 
+				isTransferValid, err := r.validateHyperlaneTransfer(ctx, transfer)
```

Comment:
`{"level":"error","ts":1731035755.109063,"caller":"hyperlane/relayer_runner.go:88","msg":"error relaying pending hyperlane transfer","error":"processing message on domain 42161: processing message on destination mailbox: execution reverted: !threshold","sourceChainID":"osmosis-1","txHash":"9C4EDA7213A3909E7F852B4047BFE74F76CD2477C8C92CA1975DD70B74547F73","stacktrace":"[github.com/skip-mev/go-fast-solver/hyperlane.(*RelayerRunner).Run](http://github.com/skip-mev/go-fast-solver/hyperlane.(*RelayerRunner).Run)\n\t/solver/hyperlane/relayer_runner.go:88\nmain.main.func6\n\t/solver/cmd/solver/main.go:153\[ngolang.org/x/sync/errgroup.(*Group).Go.func1](http://ngolang.org/x/sync/errgroup.(*Group).Go.func1)\n\t/root/.cache/go-build/golang.org/x/sync@v0.7.0/errgroup/errgroup.go:78"}`

fix for above error log, where an order was created with an invalid recipient

Reply:
you can see the recipient in this tx is set to 0 

https://www.mintscan.io/osmosis/tx/9C4EDA7213A3909E7F852B4047BFE74F76CD2477C8C92CA1975DD70B74547F73?height=23636639